src/renderer/src/assets/mobile-page.css

```
@@ -2102,3 +2129,32 @@
     font-size: 17px;
   }
 }
+
+/* Why: the app sidebar narrows this page without changing viewport media
+   queries; keep this after viewport rules so real page width wins. */
+@container (max-width: 1080px) {
+  .mobile-page-root .mp-hero {
+    --mp-flow-card-padding: 20px;
+    grid-template-columns: 1fr;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)